ViAn/Filehandler/filehandler.cpp

```diff
@@ -222,12 +226,19 @@ FH_ERROR FileHandler::delete_project(Project* proj){
  * Add a video filepath to a given project.
  * Creates Video object which is accessed further by returned id.
  */
-void FileHandler::add_video(Project* proj, std::string filePath){
+ID FileHandler::add_video(Project* proj, std::string filePath){
     Video* v = new Video(filePath);
-    proj->add_video(v);
-    this->add_file(filePath);
+    return proj->add_video(v); // video id set in proj->add_video
+}
+/**
+ * @brief FileHandler::remove_video
+ * @param proj_id
+ * @param vid_id
+ */
+void FileHandler::remove_video(ID proj_id, ID vid_id){
+    Project* proj = this->get_project(proj_id);
+    proj->remove_video(vid_id);
 }
```

Comment:
Although you find this method self-explanatory you should still comment what it does. A sentence is enough. Something like "Removes the search path to the video in the project" or something.